apps/src/aichat/views/chatWorkspace.module.scss

```diff
@@ -27,10 +27,18 @@ $default-spacing: 16px;
   background-color: #eaebeb;
 }
 
-.tabPanels {
+.tabPanelsChatWithModelActive {
```

Comment:
Why does this class not need the `display: flex` and `height: 100%` in its inner `div`?

Reply:
Ahh...when I included 
```
  > div {
    display: flex;
    height: 100%;
  }
```
The `hidden` attribute was overridden and the content from both tab panels were displayed.


https://github.com/user-attachments/assets/bf766f48-6fae-45cb-8e68-15766a50d992

But I overlooked that now when the 'Test student model' tab is active, auto-scrolling no longer works.

Reply:
We chatted offline and now using the attribute selector `hidden` to address this. Will updated - thanks so much!